api/logs/views.py

```diff
@@ -220,7 +221,7 @@ class NodeLogContributors(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin,
     required_read_scopes = [CoreScopes.USERS_READ]
     required_write_scopes = [CoreScopes.NULL]
 
-    serializer_class = UserSerializer
+    serializer_class = UnregisteredContributorSerializer
```

Comment:
Okay, I didn't know the bit about needing to be anonymizable.

As a reminder this is for the contributors_added log, and we need to get the names of the contributors added.   In the params, the contributors are a list.  So I have an array of contributors and I need to get their names.  I just changed it to stick their names under params but I don't know how to use the User Serializer there.

The only way I know how to use the User Serializer is to revert back to what it was before, with it's own view, own endpoint, etc.